SUPPORT.md

```diff
@@ -0,0 +1,34 @@
+# Commercial Support for CloudNativePG
+
+CloudNativePG is an independent open-source project and does not officially
+endorse any specific company or service provider.
+
+However, to assist users in finding professional support, the
+"[Commercial Support](https://cloudnative-pg.io/support/)"
+page offers an alphabetical list of companies and individuals providing
+CloudNativePG-related products or services.
+
+*Please note that the CloudNativePG authors are not responsible for the accuracy
+or content provided by the listed companies or individuals.*
+
+## How to Get Listed
+
+To have your company or personal services featured on this list, please submit
+a [pull request to the CloudNativePG website](https://github.com/cloudnative-pg/cloudnative-pg.github.io)
+by adding a `.md` file in the [`content/support` folder](https://github.com/cloudnative-pg/cloudnative-pg.github.io/tree/main/content/support)
+containing the following information:
+
+1. **Organisation Name**: Clearly specify the name of your company or entity.
+2. **Organisation Logo**: Provide your company logo in SVG format.
+3. **Website Link**: Include a link to your homepage or a dedicated landing
+  page that explicitly mentions CloudNativePG support and includes at least one
+  link back to [cloudnative-pg.io](https://cloudnative-pg.io).
+4. **Organisation Description**: Submit a concise description of your company,
+  with a particular focus on your CloudNativePG and Kubernetes-related
+  professional support services (*maximum 400 characters*).
```

Comment:
We could remove this fourth item and just leave organisation name, logo and link. what do other @cloudnative-pg/maintainers think? This would also imply that we remove ", or if your
claims cannot be objectively verified" below.

Reply:
I second the proposal to remove the description field. Keeping the listing streamlined with just the name, logo, and URL would simplify the process and make it more efficient, ensuring consistency across entries.